src/algebra/big_operators/intervals.lean

```diff
@@ -164,4 +166,119 @@ by rw [← sum_range_id_mul_two n, nat.mul_div_cancel]; exact dec_trivial
 
 end gauss_sum
 
+end generic
+
+section nat
+
+variable {β : Type*}
+variables (f g : ℕ → β) {m n : ℕ}
+
+-- The partial sums (starting from `0`) of `f` and `g` respectively.
+local notation `F` n:80 := ∑ i in range n, f i
+local notation `G` n:80 := ∑ i in range n, g i
+
+section monoid
+variable [add_comm_monoid β]
+
+lemma sum_Ico_empty : (∑ i in Ico n n, f i) = 0 :=
+begin
+  convert sum_empty,
+  exact Ico_self n,
+end
```

Comment:
It looks like this lemma and the one below can be proved by simp, or just a sequence of rewrites, so perhaps they don't need to be their own lemmas?

Reply:
I need to rewrite `f n` to `sum (Ico m (m+1)) f` when proving another lemma, hence why `sum_Ico_single` exists. I think I stuck this one in for completion's sake. If I were to remove both of them, I would have to inline `sum_Ico_single` in the lemma that depends on it, or find another way to write the proof. Is this what you want?

Reply:
These have been removed.